tmt/convert.py

```diff
@@ -850,7 +850,19 @@ def read_polarion_case(
 
     # Update description
     if polarion_case.description:
-        current_data['description'] = str(polarion_case.description)
+        description = str(polarion_case.description).replace('<br/>', '\n')
+        if 'Environment variables' in description:
+            description, envvars = description.split('Environment variables:', maxsplit=1)
+            if not current_data.get('environment'):
+                current_data['environment'] = {}
+            for envvar in envvars.split('\n'):
+                if not envvar:
```

Comment:
1. `splitlines()` might be better than `split('\n')` as it supports wide variety of end-of-line characters.
2. is there a chance of encountering lines that are not empty, but contain only whitespace? I'd recommend adding `envvar = envvar.strip()` before checking whether the `envvar` is false-ish.